apps/pdl-forvalter/src/main/java/no/nav/pdl/forvalter/utils/PostnummerService.java

```diff
@@ -0,0 +1,30 @@
+package no.nav.pdl.forvalter.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+@Slf4j
+@Service
+public class PostnummerService {
+
+    private static final Properties postnumre = new Properties();
+
+    static {
+        var resource = new ClassPathResource("postnummer/postnummer.yaml");
```

Comment:
Bruke et rammeverk for å lese yaml?

EKs jackson-dataformat-yaml som er tilgjenelig fra i spring boot 


`implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'`